### ISSUE TITLE
Support importing lists

### ISSUE THREAD

**Issue by mfcndw1** [tootsuite/mastodon#15015]
### Pitch

Currently only exporting lists are available, but not importing list. It would be great that people can also import their lists .csv to new account.

### Motivation
Make migrating account smoother and more organized.

**Comment by quinncomendant** [comment]
I came here to suggest the same: I have a CSV of users that I want to add to one of my lists without following them first. The current import tool has a Import Type of `Following list` but not a way to add them only to a list. The Import Type menu should include `Custom list` and then allow choosing one of my lists to select.

**Comment by fadookie** [comment]
I have made a command-line tool that can import Mastodon lists into a different instance. Hopefully this will be added into the actual app, but it was faster to make an external tool. If you are interested in using this and would like a web UI, [please let me know](https://github.com/fadookie/masto-list-importer/issues/1). 

https://github.com/fadookie/masto-list-importer

**Comment by kerim** [comment]
Please implement this. I just migrated and was shocked that after spending so much time curating my lists they were just lost!!!

**Comment by JaGeo** [comment]
This tool would also allow to easily edit externally and then import. I am currently following 1000 people and have no chance of sorting this via the webinterface. I would spend hours on this.

**Comment by electricbrick** [comment]
Agreed. Fortunately my lists weren't considerably curated, but I find it distressing that most users will just *lose* lists without being aware of it if they migrate on a whim.

@fadookie your list importer was helpful and (mostly*) straightforward. BIG THANK YOU! 

*It wasn't immediately clear to me that I needed to include the full instance URL including the https — maybe consider adding that as a description attribute for the instance input field?

**Comment by AmbularD** [comment]
It's hard to understand why the ability to export lists is included without a corresponding import feature.  This would seem to be a necessary function.

**Comment by evolvingKirt** [comment]
> Please implement this. I just migrated and was shocked that after spending so much time curating my lists they were just lost!!!

lists were the first thing i thought of copying when i saw you coyote download your data

i wish i could copy my lists out if twitter, but even copying within mastadon would be good, to share wuthering friends or just for migrating between servers
